tests/migration/openldap_to_389ds.pm

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Implement opendlap to 389ds migration tools test case into openQA
+#
+# Maintainer: wegao <wegao@suse.com>
+
+use base 'consoletest';
+use testapi;
+use strict;
+use warnings;
+use utils;
+use lockapi;
+use Utils::Systemd qw(systemctl disable_and_stop_service);
+use registration qw(add_suseconnect_product);
+
+sub run {
+    select_console("root-console");
+
+    # Install 389-ds and sssd on client
+    zypper_call("in 389-ds sssd sssd-tools");
+    script_run "rpm -qa 389-ds";
+    # Install openldap since we need use slaptest tools
+    add_suseconnect_product('sle-module-legacy');
+    zypper_call("in openldap2 sssd-ldap sssd-tools openldap2-client");
```

Comment:
you install sssd_tools again here. is this intentional?

Reply:
Yes, this package will install clear cache tools for sssd